src/server/shared/DataStores/DBCDatabaseLoader.cpp

```diff
@@ -93,6 +93,7 @@ char* DBCDatabaseLoader::Load(uint32& records, char**& indexTable)
                     break;
                 case FT_SORT:
                 case FT_NA:
+                case FT_NA_BYTE:
```

Comment:
It looks like this line is in a generic method that's used for all DBC tables. We should make sure to test that no existing DBC functionality was broken

Reply:
@steenburgh Feel free to test the FT_NA and FT_NA_BYTE are both unused nothing should break dbc functionality.

https://github.com/azerothcore/azerothcore-wotlk/blob/4b7454574412cfe23bb5902b9b3d42d2313b3541/src/common/DataStores/DBCFileLoader.h#L17